src/blackcore/fsd/fsdclient.cpp

```diff
@@ -118,7 +118,7 @@ namespace BlackCore::Fsd
         connect(&m_scheduledConfigUpdate, &QTimer::timeout, this, &CFSDClient::sendIncrementalAircraftConfig);
 
         m_fsdSendMessageTimer.setObjectName(this->objectName().append(":m_fsdSendMessageTimer"));
-        connect(&m_fsdSendMessageTimer, &QTimer::timeout, this, &CFSDClient::sendQueuedMessage);
+        connect(&m_fsdSendMessageTimer, &QTimer::timeout, this, [this]() { this->sendQueuedMessage(); });
```

Comment:
What is the reason for using a lambda here?

Reply:
I tried with ``qOverload<>(&CFSDClient::sendQueuedMessage)`` first. But this does not compile as the other overloaded message is a template function. Maybe I am missing something?

Reply:
Curious why that would not work. Never mind, it is a minor annoyance.